src/containers/Posts/FullPost/FullPost.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import './FullPost.css';
+import { connect } from 'react-redux';
+import * as actions from '../../../store/actions/index';
+import Modal from '../../../components/UI/Modal/Modal';
+import UpdatePost from '../updatePost/updatePost';
+import { Card } from 'semantic-ui-react';
+import Spinner from '../../../components/UI/Spinner/Spinner';
+
+class FullPost extends Component {
+    state = {
+        updateModal: false
+    }
+
+    componentDidMount() {
+        this.props.getPostById(this.props.token, this.props.userId, this.props.match.params.id);
+    }
+
+    deletePostHandler = () => {
+        this.props.deletePost(this.props.token, this.props.userId, this.props.match.params.id);
+        setTimeout(() => {
```

Comment:
Do not use `setTimeout` you can use `async` and `await`

Reply:
working on this